### DISCUSSION TITLE
Instant loading is no restoring the page

### DISCUSSION THREAD

**Discussion by mercenaruss** [squidfunk/mkdocs-material#5815] [Q&A] [answered]
I have a custom part of page what include some JavaScript what is loading shopping cart.
If i open the page directly Javascript is loading and i see the cart and shopping options.
If i change page and return back to product page, shopping cart is not visible anymore .Only full page refresh is helping.

Here is minimal reproduction:

[example.zip](https://github.com/squidfunk/mkdocs-material/files/12275006/example.zip)

Any idea how to keep instant loading and in the same time to have shopping cart working?

**Comment by squidfunk** [comment, marked as answer]
Duplicate of #5816
